sopel/modules/units.py

```diff
@@ -56,7 +56,11 @@ def temperature(bot, trigger):
 
     kelvin = c_to_k(celsius)
     fahrenheit = c_to_f(celsius)
-    bot.reply("{:.2f}°C = {:.2f}°F = {:.2f}K".format(celsius, fahrenheit, kelvin))
+
+    if kelvin >= 0:
+        bot.reply("{:.2f}°C = {:.2f}°F = {:.2f}K".format(celsius, fahrenheit, kelvin))
+    else:
+        bot.reply("Physically impossible temperature. Do you live in an imaginary world?")
```

Comment:
I'm happy with everything except the high level of sass in this error.

Reply:
Sopel is sassy by default